docs/recipes/grunt-ava-parallel-tests.md

```diff
@@ -0,0 +1,56 @@
+Multi level parallel tests runs where the time taken is only in the order of your longest running test !
+
+##Sample Test harness
+https://github.com/subash-canapathy/grunt-ava-parallel-tests
+
+You can use this as a bootstrap test runner / harness to build automated test framework on. It is designed with webdriver/appium end to end tests in mind so that you can run your UI tests with only your grid as the limiting factor.
+## Usage
```

Comment:
You should have a blank like before and after Markdown headings (you do elsewhere, so this is more about consistency).